src/frontend/components/Felleskomponenter/JaNeiSpm/JaNeiSpm.tsx

```diff
@@ -65,7 +65,7 @@ const JaNeiSpm: React.FC<IJaNeiSpmProps> = ({
             <JaNeiSpørsmål
                 {...felt.hentNavInputProps(skjema.visFeilmeldinger)}
                 initiellVerdi={felt.verdi}
-                name={guid()}
+                name={uuidv4()}
```

Comment:
Er det riktig å bruke uuid/guidx her? 🤔

Reply:
Hva brukes name til?

Reply:
Ser ut som det blir brukt av RadioGroup for å override noe internal name: https://aksel.nav.no/designsystem/komponenter/radio/kode, men hvorfor man trenger det er jeg litt usikker på

Reply:
Ser ut som designsystemet setter dette til en tilfeldig id, så da burde uuid funke 🙌
<img width="576" alt="image" src="https://user-images.githubusercontent.com/17828446/193255784-e1f14152-011e-42f8-a9c9-e487ec87d081.png">
<img width="1053" alt="image" src="https://user-images.githubusercontent.com/17828446/193255897-c707dc69-1725-41be-b3bc-a1595213289e.png">